examples/electron_phonon/run_electron_phonon.py

```diff
@@ -0,0 +1,24 @@
+from pyvotca import Molecule
+from pyvotca import Orca
+from pyvotca import Electronphonon
+
+
+# define a molecule
+mol=Molecule()
+
+# load xyz 
+mol.read_xyz_file('./el-ph/NPB.xyz')
```

Comment:
it should be `mol.read_xyz_file('./NPB.xyz')`

Reply:
True x3